### PR TITLE
release_notes - correctly state that nightly is a debug release

### PR DESCRIPTION
**Pull request by nerdCopter**
* nightly builds currently and incorrectly state `This is a release build. It does not contain the debug console.`
* this PR changes it to `This is a debug build. It contains the debug console.`

image

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by haslinghuis — APPROVED** [review]

**Review by ctzsnooze — APPROVED** [review]

**Review by blckmn — APPROVED** [review]

**Pull request merged**
